scripts/update/Updater.php

```diff
@@ -100,6 +102,16 @@ public function update($initialVersion)
         }
         
         $this->skip('1.2.2', '1.6.1');
+
+        if ($this->isVersion('1.6.1')) {
+
+            AclProxy::applyRule(new AccessRule('grant', INSTANCE_ROLE_ANONYMOUS,
+                ['ext'=>'taoCe', 'mod' => 'Main', 'act' => 'rootEntry']));
+            AclProxy::applyRule(new AccessRule('grant', INSTANCE_ROLE_BASEUSER,
```

Comment:
Same remark as above, 

"Actually, BaseUserRole includes AnonymousRole. So basically, I think there is no necessity to explicitly grand the BaseUserRole, but just the AnonymousRoule. Indeed, BaseUserRole will implicitly inherits AnymousRole grants."